R/get_slf_lookup_paths.R

```diff
@@ -53,3 +53,22 @@ get_slf_deaths_path <- function(update = latest_update(), ...) {
 
   return(slf_deaths_file_path)
 }
+
+
+#' Get the full path to the SLF Care Home lookup
+#'
+#' @param update the update month (defaults to use \code{\link{latest_update}})
+#' @param ... additional arguments passed to \code{\link{get_file_path}}
+#'
+#' @return The path to the SLF Care Home lookup as an \code{\link[fs]{path}}
+#' @export
+#' @family file path functions
+#' @seealso \code{\link{get_file_path}} for the generic function.
+get_slf_ch_path <- function(update = latest_update(), ...) {
+  get_file_path(
+    directory = fs::path(get_slf_dir(), "Lookups"),
+    file_name = glue::glue("Care_Home_Lookup_All.xlsx"),
+    check_mode = "write",
+    ...
+  )
+}
```

Comment:
Can we rename this to `get_slf_ch_name_lookup_path` (or similar) please? So it's not confused with any CH data, i.e. make it clear it's the name lookup.